### DISCUSSION TITLE
有个setting.py和.env的配置问题

### DISCUSSION THREAD

**Discussion by peng19832** [hhyo/Archery#2032] [Q&A]
1.9.1版本，咨询下setting.py和local_setting.py的，覆盖逻辑：
从1.8版本升级的话，是复用1.8的setting.py，还是只需要保持秘钥就可以？
如果，直接复用1.8的，那.env还有作用不？

**Comment by LeoQuote** [comment]
看这里 https://github.com/hhyo/Archery/pull/1543 

写了优先级

建议是:
原有的 settings.py 里的修改, 全部回滚, 然后自己根据需要去修改, 能走环境变量, 或者 .env 文件的, 走 环境变量或 .env 文件
走不了环境变量的, 走 local_settings.py

**Reply by peng19832** [reply]
@LeoQuote 
大佬，帮忙看下这是为什么呢？
我测试CSRF_TRUSTED_ORIGINS，配置在.env中，一直报
![image](https://user-images.githubusercontent.com/9590209/211985774-6a6453ac-3cce-4e3d-831e-fb3f986351ac.png)
![image](https://user-images.githubusercontent.com/9590209/211971639-5eae412a-d0d0-491d-8097-aa48e777f0bc.png)

换到setting.py里，不再报这个，错误变成这个了：
![image](https://user-images.githubusercontent.com/9590209/211986637-060907bc-c059-4006-956a-c84303d522f0.png)

搞不明白为啥，我最后只有注释掉这个中间件，临时跑起来了：
![image](https://user-images.githubusercontent.com/9590209/212631460-1faf75df-4692-4634-a383-d14051010ac8.png)

**Reply by peng19832** [reply]
噢，需要清空浏览器cookies
